docs/components/pagination.md

```diff
@@ -11,275 +11,298 @@
 A total number of items inside the paginated data source. This prop is **required**.
 
 ```html
-<KPagination :totalCount="50"/>
+<KPagination :total-count="50"/>
 ```
 
 ### pageSizes
 
-A list of page sizes that the user can choose from.
+An array of numbers of page sizes that the user can choose from. The default page sizes are: `[15, 30, 50, 75, 100]`.
 
-Prop is an Array of numbers. The default page sizes are: `[15, 30, 50, 75, 100]`.
+You can provide custom page sizes. The first one in the array will be the initial page size unless otherwise specified through [`initialPageSize` prop](#initialpagesize).
```

Comment:
```suggestion
You can provide custom page sizes. The first number in the array will be the initial page size unless otherwise specified through the [`initialPageSize` prop](#initialpagesize).
```